tests/test_appdata.py

```diff
@@ -46,13 +46,90 @@ def test_simple(self):
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <releases>
-    <release version="4.5.6" date="2020-02-02"/>
+    <release version="4.5.6" date="2020-02-02">
+      <description></description>
+    </release>
     <release version="1.2.3" date="2019-01-01"/>
   </releases>
 </component>
             """.strip(),
         )
 
+    @unittest.expectedFailure
+    def test_four_space_no_releases_element(self):
+        # FIXME: This ends up indenting <releases> correctly, but
+        # <release> and <description> incorrectly get the default 2-space
```

Comment:
Oh I'd forgotten about this one. I think it's such an edge case that we can accept the failure for now.